src/font.rs

```diff
@@ -67,8 +67,10 @@ impl<'a> Font<'a> {
     ///  - `text`: text to render.
     ///  - `row`: x (Left/Right align) or y (Up/Down align) offset where to stop
     ///    rendering.
-    ///  - `wh`: the size of each character in X & Y dimensions.
```

Comment:
This parameter doesn't seem to be there. Maybe it's leftover docs from a previous version.

Reply:
Yeah, this got replaced by using Transforms from the footile crate, so it shouldn't be there anymore.